### DISCUSSION TITLE
Add OSC outputs

### DISCUSSION THREAD

**Discussion by gmschroeder** [brunoherbelin/vimix#134] [General]
I'm using external software to convert midi from a controller to OSC. I'm trying to enable the color correction programmatically.

Vimix sending an OSC message when it loads/says it's ready in the log window/whatever a source would make that much easier. I have 'duh' moments with this constantly.

**Comment by brunoherbelin** [comment]
Thanks! Indeed, this could be easily implemented. I just didn't want to overload with unread OSC messages, but if it can be useful, improving the two-ways OSC communication can be done !!

What else than in the messages sent by vimix do you suggest ? 
https://github.com/brunoherbelin/vimix/wiki/Open-Sound-Control-API#messages-sent-by-vimix
